File: react-js/Micro-frontend para uma empresa promotora de crédito/packages/core/src/plataforma-cliente/notificacoes/notificacao.model.ts
export enum NotificacaoSeveridadeEnum {
  baixa = 0,
  media = 1,
  urgente = 2,
  gravissima = 3,
}

export interface Notificacao {
  id: number;
  titulo?: string;
  descricao?: string;
  urlReferencia?: string;
  dataNotificacao: string;
  dataVisualizacao: string;
  severidade: NotificacaoSeveridadeEnum;
}

File: net-core-apps/Monolito para uma promotora de crédito/Aplicacao/Model/Notificacao/NotificacaoModel.cs
using System;
using Dominio.Enum.Notificacoes;

namespace Aplicacao.Model.Notificacoes
{
    public class NotificacaoModel
    {
        public int Id { get; set; }
        public string Titulo { get; set; }
        public string Descricao { get; set; }
        public string UrlReferencia { get; set; }
        public DateTime DataNotificacao { get; set; }
        public DateTime? DataVisualizacao { get; set; }
        public NotificacaoSeveridade Severidade { get; set; }
    }
}
